src/content/pages/explore.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: More Tips on Exploring Prague
+subtitle: In EuroPython there are many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surrounding area?
+---
+
+import { Image } from "astro:assets";
+
+import venueImage from "./images/prague.jpg";
+
+# More Tips on Exploring Prague
+
+<Image src={venueImage} width="1200" alt="" />
+
+<br></br>
+
+At EuroPython many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surrounding area?
+
+Here are some tips and things to do provided by our volunteers, hope it helps you and your companions to have a great time in Prague:
+
+## Traveling around
+
+### Public transport
+
+In Prague the recommended way to get around is public transport:
+
+- There’s some info [here](https://ep2024.europython.eu/where#getting-around-in-prague)
+- You can use your normal card to travel around by train within Prague
+- Usually fast and reliable
+- After midnight the metro stops. - But there are buses
+- Mostly safe, no no-go stations nor anything similar
+
+### Do I need to hire a car?
+
+Some people may consider hiring a car. But:
+
+- In Prague this is unnecessary, it’s hard to find parking, and it’s expensive
+- You can hire a car at the airport or other places in the city (like you normally do in other cities)
+- If you want to visit other towns, you can use trains as well (they’re mostly cheap)
+
+### Be prepared for the weather
+
+If you want to explore the city, you will be exposed to the weather. So here some answers to frequent questions like: What’s the weather like? What to carry with you?
+
+- Usually 30°C+, so carry water
+- There are no water fountains nor public toilets, so carry more water
+- Use Sunscreen
+- It rarely rains, but when it does it’s shower rain - it’s best to just hide somewhere until it stops (it usually stops after half an hour or so).
+
+## What to do while in Prague?
+
+There are plenty of thing you can do in Prague. Depending on your preference, you can find things that interests you and your group.
+
+### For culture lovers
+
+- Summer cinema
+- Theater festival Za dveřmi 2024
+
+### For art lovers
+
+- DOX
+- Soví mlýny
+- National museum
+- Jazz dock for live jazz on a boat
+- Žižkov area for bars and pubs
```

Comment:
bars and pubs are lovely, but I am not sure why they are put in the art lovers' section. Do these bars and pubs also function as galleries or host gigs?

Reply:
Jazz dock hosts jazz concerts every night. About žižkov area - there are some pubs and bars with live music as well, but I agree that maybe it's better to move it somewhere else.

Reply:
Yeah I think the jazz dock is very relevant here (and great shout!). For žižkov, maybe mention something like "many of which will host live music".